Core/Core/View/Base/Webview/WebView.swift

```diff
@@ -145,20 +145,23 @@ public struct WebView: UIViewRepresentable {
             }
             
             let baseURL = await parent.viewModel.baseURL
-            if !baseURL.isEmpty, !url.absoluteString.starts(with: baseURL) {
-                if navigationAction.navigationType == .other {
-                    return .allow
-                } else if navigationAction.navigationType == .linkActivated {
-                    await MainActor.run {
+            switch navigationAction.navigationType {
+            case .other, .formSubmitted:
```

Comment:
In this case, do we need to handle `formResubmitted` as well? or it's fine to through it to default?

Reply:
by `default` for not baseURL `.cancel` is returning for any type.

Reply:
`.allow` was added for type `.other` and `.formSubmitted` - I don't know the reason. But do you think it's good idea to change this now?

Reply:
let's keep this for a while, but I would say if we are adding `.formSubmitted` we should also add `.formResubmitted`, thoughts?

Reply:
Perhaps we should. But we need to have some request from QA for this when something doesn't work. I mean current state (with only `.other` and `.formSubmitted` for baseURL) was tested and no issues was found. Like `if it ain't broke, don't fix it` you know.